main/pipeline/pipeline.cc

```diff
@@ -174,12 +174,25 @@ const unique_ptr<parser::Node> convertPrismToSorbet(pm_node_t *node, pm_parser_t
         }
         case PM_PROGRAM_NODE: {
             pm_program_node *programNode = reinterpret_cast<pm_program_node *>(node);
-            pm_statements_node *stmts = programNode->statements;
 
-            auto size = stmts->body.size;
+            return convertPrismToSorbet(reinterpret_cast<pm_node *>(programNode->statements), parser, gs);
```

Comment:
Wow GitHub chose the most confusing way to display this diff. I have not changed anything about the handling of `PM_RATIONAL_NODE`. I've moved most of the logic that was under `PM_PROGRAM_NODE` into `PM_STATEMENTS_NODE`.